src/crypto/crypto.js

```diff
@@ -123,7 +123,7 @@ export default {
         const d = key_params[2].toBN(); // de = 1 mod (p-1)(q-1)
         const p = key_params[3].toBN();
         const q = key_params[4].toBN();
-        const u = key_params[5].toBN(); // q^-1 mod p
+        const u = key_params[5].toBN(); // p^-1 mod q, as per https://tools.ietf.org/html/rfc4880#section-5.5.3
```

Comment:
This link seems to apply more generally to this entire function. Perhaps add it in the comment above it instead, or change the existing link as this one seems to be more applicable? I.e.
```diff
-   * See {@link https://tools.ietf.org/html/rfc4880#section-9.1|RFC 4880 9.1} for public key algorithms.
+   * See {@link https://tools.ietf.org/html/rfc4880#section-5.5.3|RFC 4880 5.5.3}.
```